build.sbt

```diff
@@ -80,6 +80,7 @@ lazy val admin = project
   .in(file("."))
   .settings(testSettings, buildInfoSettings)
   .enablePlugins(BuildInfoPlugin, ServicePackagingPlugin)
+  .dependsOn(client)
```

Comment:
I think we decided not to have dependencies between the service and the client

Reply:
That would effectively mean that you would have to duplicate the Event classes.
Note that we do the same for `AccessControlLists` and so on.